src/Core/System/Snippet/Command/Util/CountryAgnosticFileValidator.php

```diff
@@ -0,0 +1,175 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\Snippet\Command\Util;
+
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\System\Snippet\SnippetException;
+use Shopware\Core\System\Snippet\SnippetPatterns;
+use Shopware\Core\System\Snippet\Struct\TranslationFile;
+use Shopware\Core\System\Snippet\Struct\TranslationFileCollection;
+use Shopware\Core\System\Snippet\Struct\ValidatedTranslationFileOptions;
+use Shopware\Core\System\Snippet\Struct\ValidatedTranslationFileStruct;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @internal
+ */
+#[Package('discovery')]
+class CountryAgnosticFileValidator
+{
+    public const CORE_DOMAINS = [
```

Comment:
is core the right wording? What do you think about SHOPWARE_DOMAINS or PLATFORM_DOMAINS?